src/inference/include/openvino/runtime/intel_gpu/properties.hpp

```diff
@@ -123,6 +123,14 @@ static constexpr Property<int64_t> available_device_mem{"AVAILABLE_DEVICE_MEM_SI
  * @ingroup ov_runtime_ocl_gpu_prop_cpp_api
  */
 static constexpr Property<bool> enable_sdpa_optimization{"GPU_ENABLE_SDPA_OPTIMIZATION"};
+
+/**
+ * @brief Turning on this property enables kernels reuse between implementations, resulting in a lower memory footprint.
+ * However, as a drawback, set_arguments() call must be made for each enqueue() call.
```

Comment:
what does it mean for users? Runtime overhead and slower execution?

Reply:
@ilya-lavrenov , yes, that's right. I slightly updated the description to make it more obvious